ingest/ingest.go

```diff
@@ -146,6 +146,10 @@ func (i *FileIngest) InsertData(c *fiber.Ctx) error {
 	if !ok {
 		writer = NewFileWriter(
 			dir,
+			i.Config.Ingest.MaxAgeSeconds,
```

Comment:
These 4 items are already available in `i.Config` which we're passing in on line 153